icu4c/source/test/intltest/utfiteratortest.cpp

```diff
@@ -680,6 +648,8 @@ class UTFIteratorTest : public IntlTest {
         const std::u16string codeUnits = u"𒀭𒊺𒉀​𒍠𒊩";
         static_assert(std::ranges::contiguous_range<decltype(codeUnits)>);
         auto codePoints = utfStringCodePoints<char32_t, UTF_BEHAVIOR_FFFD>(codeUnits);
+        auto end = codePoints.end();
+        static_assert(std::sentinel_for < decltype(end), decltype(codePoints.begin())>);
```

Comment:
@eggrobin Is this the debug assertion that you want to remove?
Is it useful to keep? (Except remove the spaces around `<` ?)

Reply:
Oh right, I forgot to push that commit…